commands/cluster/cluster.go

```diff
@@ -17,35 +17,113 @@ package commands
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/spf13/cobra"
 
+	"github.com/hazelcast/hazelcast-commandline-client/config"
+	hzcerror "github.com/hazelcast/hazelcast-commandline-client/errors"
 	"github.com/hazelcast/hazelcast-commandline-client/internal"
 )
 
-const invocationOnCloudErrorMessage = "Cluster operations on cloud are not supported. Checkout https://github.com/hazelcast/hazelcast-cloud-cli for cluster management on cloud."
-
-var ClusterCmd = &cobra.Command{
-	Use:   "cluster {get-state | change-state | shutdown | query} [--state new-state]",
-	Short: "administrative cluster operations",
-	Long:  `administrative cluster operations which controls a Hazelcast cluster by manipulating its state and other features`,
-	RunE: func(cmd *cobra.Command, args []string) error {
-		hzConf, err := internal.MakeConfig()
-		if err != nil {
-			return err
-		}
-		// check if it is cloud invocation
-		if hzConf.Cluster.Cloud.Token != "" {
-			fmt.Println(invocationOnCloudErrorMessage)
+const invocationOnCloudInfoMessage = "Cluster operations on cloud are not supported. Checkout https://github.com/hazelcast/hazelcast-cloud-cli for cluster management on cloud."
+
+func New() *cobra.Command {
```

Comment:
The definition of `New` explains too many context. So we can renamed it in a more precise way.

Reply:
`NewCmd` makes more sense I guess. Then the usage will be `cluster.NewCmd()`. WDYT ?

Reply:
I think the problem is with the name of the package. If it is renamed to `clustercmd`, then `clustercmd.New` sounds natural.

Reply:
Also we may remove the `commands` folder and move `clustercmd`, etc. to project root.